### DISCUSSION TITLE
Deploy backend on Vercel, it is possible ?

### DISCUSSION THREAD

**Discussion by offlink-io** [medusajs/medusa#4377] [Q&A] [answered]
Hi,

It is possible to deploy the **backend** to Vercel ?
Some examples with Heroku or Digital Ocean are provide, but nothing about Vercel (https://docs.medusajs.com/deployments/server/).

Thx :)

**Comment by kleenkanteen** [comment]
Not supported at the moment afaik. Railway is just as easy though, I recommend that.

**Comment by srindom** [comment, marked as answer]
Hi @la-ruee-vers-le-bois - you can't deploy the entire backend to Vercel. However, our new Product Module enables Vercel deployments for a subset of backend functionality. We are working on more modules that can be deployed like this :)

You can check out the docs for the product module here: https://docs.medusajs.com/modules/products/serverless-module
